meta-ostro/recipes-core/vboxmanage/vboxmanage-native_5.0.16.bb

```diff
@@ -0,0 +1,73 @@
+LICENSE="GPLv2"
+LIC_FILES_CHKSUM="file://COPYING;md5=e197d5641bb35b29d46ca8c4bf7f2660"
+
+inherit native
+
+BPN="VirtualBox"
+
+DEPENDS="libxml2-native glib-2.0-native libidl-native"
+
+SRC_URI=" \
+    http://download.virtualbox.org/virtualbox/${PV}/VirtualBox-${PV}.tar.bz2 \
+    file://xpidl-build-fixes.patch \
+    "
+
+# export path to native libdir, as this is needed during build
+# (see xpidl build fix patch)
+export VIRTUALBOX_YOCTO_LIBDIR="${STAGING_LIBDIR_NATIVE}"
+
+SRC_URI[md5sum] = "1752a485b1cb377cee5f196918cda741"
+SRC_URI[sha256sum] = "f5a44d33a1db911f445b2eb2d22d9293a96a535cba488b5a235577ef868fa63c"
+
+# Disable everything we do not need (which turns out to be almost everything)
+EXTRA_OECONF = " \
+    --disable-hardening \
+    --disable-python \
+    --disable-java \
+    --disable-vmmraw \
+    --disable-sdl-ttf \
+    --disable-alsa \
+    --disable-pulse \
+    --disable-dbus \
+    --disable-kmods \
+    --disable-opengl \
+    --disable-extpack \
+    --disable-docs \
+    --disable-libvpx \
+    --disable-udptunnel \
+    --disable-devmapper \
+    --build-headless \
+    "
+
+do_configure() {
+    ./configure ${EXTRA_OECONF}
```

Comment:
After reading the doc (http://www.yoctoproject.org/docs/2.1/mega-manual/mega-manual.html#new-recipe-configuring-the-recipe), only autotools or cmake based projects handle do_configure() automatically. As this is neither, we must handle configuration manually. 

I added a comment to clarify this.